extensions/amp-orientation-observer/0.1/amp-orientation-observer.js

```diff
@@ -100,9 +89,17 @@ export class AmpOrientationObserver extends AMP.BaseElement {
         '`window.DeviceOrientationEvent`'
     );
 
-    this.alphaRange_ = this.parseAttributes_('alpha-range', this.alphaRange_);
-    this.betaRange_ = this.parseAttributes_('beta-range', this.betaRange_);
-    this.gammaRange_ = this.parseAttributes_('gamma-range', this.gammaRange_);
+    AXES.forEach(axis => {
+      this.range_[axis] = this.parseAttributes_(
```

Comment:
I don't think this will work with property renaming, since the properties on `this.range_` will be renamed to things like `a` and `b`.

To get around this, you could do something like:

```javascript
/** @private {!Object<string, !Array<number>>} */
this.range_ = {
  alpha: [0, 360],
};
```

Alternatively, you can do this using arrays, and use array indicies from an enum (and move the axis names into an array as well).

Reply:
Fixed using string literals, also added cloning the default value. But let me know if you think using arrays is better. I just thought it might make the code slightly less readable.